java_training/src/com/lftechnology/jan12/ArrayListMain.java

```diff
@@ -0,0 +1,94 @@
+package com.lftechnology.jan12;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * This class explores the functionalities of the {@link ArrayListManipulator} class
+ * 
+ * @author nerisa
```

Comment:
Or @utsabneupane I would suggest you to use your personal email address instead of `@lftechnology.com`.
What happens when you leave the company? :D